priv/repo/migrations/20171221060104_create_mentors.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Hades.Repo.Migrations.CreateMentors do
+  use Ecto.Migration
+
+  def change do
+    create table(:mentors) do
+      add :is_active, :boolean, default: false, null: false
+      add :max_mentorships, :integer
+      add :skill_areas, {:array, :string}
+      add :user_id, references(:users, on_delete: :nothing)
```

Comment:
If a user is deleted, a mentor record should not be deleted also?

Reply:
Yes, that would make sense.
I think we should think though if we want to allow users to be deleted in the first place, because we will lose history. Perhaps we can deactivate them or something. What do you think ?